lib/jekyll/plugin_manager.rb

```diff
@@ -82,6 +82,14 @@ def plugins_path
       if site.config["plugins_dir"].eql? Jekyll::Configuration::DEFAULTS["plugins_dir"]
         [site.in_source_dir(site.config["plugins_dir"])]
       else
+        custom_plugins_path
+      end
+    end
+
+    def custom_plugins_path
+      if Jekyll::Utils::Platforms.windows?
+        Array(site.config["plugins_dir"])
```

Comment:
What makes the `File.expand_path(d)` call invalid? I believe what needs to change is the _test_, not this code.